src/app/content/components/ContentExcerpt.spec.tsx

```diff
@@ -14,6 +14,14 @@ import { findArchiveTreeNodeById } from '../utils/archiveTreeUtils';
 import * as contentManipulation from '../utils/contentManipulation';
 import ContentExcerpt from './ContentExcerpt';
 
+jest.mock('../../../config', () => {
+  const config = jest.requireActual('../../../config');
+  return {
+    ...config,
+    REACT_APP_ARCHIVE_URL: '',
```

Comment:
we should probably deal with this in a better way. i'm realizing that the current logic is appending the query param to routes if `REACT_APP_ARCHIVE_URL` is defined at all, but in the future when adopting rap it'll always be defined....

Reply:
so like deal with it better in the mocks or everywhere?